appointment/views.py

```diff
@@ -145,4 +150,98 @@ def appointment_participants(request, meeting_id):
     """Show the participant on each appointment."""
     meeting = get_object_or_404(Meeting, pk=meeting_id)
     participants = UserMeeting.objects.filter(meeting=meeting, is_join=True)
-    return render(request, 'appointment/participants.html', {'participants': participants})
+    return render(request, 'appointment/participants.html', {'participants': participants, 'meeting': meeting})
+
+
+class CreateMeeting(LoginRequiredMixin, SuccessMessageMixin, generic.CreateView):
+    """For create a meeting"""
+    model = Meeting
+    form_class = UserCreateMeetForm
+    template_name = 'appointment/create_meeting.html'
+    success_message = "%(subject)s was created successfully!!"
+
+    def form_valid(self, form):
+        """Add host permission to user that create meeting."""
+        form.instance.host = self.request.user
+        return super().form_valid(form)
+
+    def get_success_message(self, cleaned_data):
+        """Get cleaned date form."""
+        return self.success_message % dict(
+            cleaned_data,
+            subject=self.object.subject,
+        )
+
+
+class EditMeeting(LoginRequiredMixin, UserPassesTestMixin, SuccessMessageMixin, generic.UpdateView):
+    """For edit a meeting. Only host can do this."""
+    model = Meeting
+    form_class = UserEditMeetForm
+    template_name = 'appointment/edit_meeting.html'
+    success_message = "%(subject)s was edited successfully!!"
+
+    def form_valid(self, form):
+        """Add host permission to user that create meeting."""
+        form.instance.host = self.request.user
+        return super().form_valid(form)
+
+    def test_func(self):
+        """Test the user to edit meeting. The host can editable only."""
+        meeting = self.get_object()
+        if self.request.user == meeting.host:
+            return True
+        return False
```

Comment:
Test the user permission (Is the host user?). The host user can edit meeting only